acre/core.py

```diff
@@ -44,14 +44,17 @@ def compute(env,
     # Collect dependencies
     dependencies = []
     for key, value in env.items():
-        dependent_keys = re.findall("{(.+?)}", value)
-        for dependency in dependent_keys:
-            # Ignore direct references to itself because
-            # we don't format with itself anyway
-            if dependency == key:
-                continue
+        try:
+            dependent_keys = re.findall("{(.+?)}", value)
+            for dependency in dependent_keys:
+                # Ignore direct references to itself because
+                # we don't format with itself anyway
+                if dependency == key:
+                    continue
 
-            dependencies.append((key, dependency))
+                dependencies.append((key, dependency))
+        except Exception:
+            dependencies.append((key, value))
```

Comment:
Why does this need to be a try..except? Doesn't make much sense to me.

That'd only occur if `value` is not a string maybe? E.g. a number of None? But that basically means the `env` isn't an "environment" so the input is already invalid to begin with? This **should** error. No?